docs/guide/connecting.asciidoc

```diff
@@ -321,8 +321,8 @@ es = Elasticsearch(
 ==== API Key authentication
 
 You can configure the client to use {es}'s API Key for connecting to your 
-cluster. Note that you need the values of `id` and `api_key` to
-https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html[authenticate via an API Key].
+cluster. Note that you need the API Key credential representing the Base64-encoding of the id and api_key value 
+https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html[generated from the API Key].
```

Comment:
This looks good and I don't want to delay this pull request further, but for future work I think we should encourage users going to Kibana instead which has a clear UI and docs (https://www.elastic.co/guide/en/kibana/current/api-keys.html) and displays the correct encoding by default:

![image](https://github.com/elastic/elasticsearch-py/assets/42327/895d2656-3272-4767-aaf2-65d6d1608a43)

Reply:
Here's the result: https://elasticsearch-py_bk_2477.docs-preview.app.elstc.co/guide/en/elasticsearch/client/python-api/master/connecting.html#auth-apikey.

I don't understand the sentence:

> You can configure the client to use Elasticsearch’s API Key for connecting to your cluster. Note that you need the API Key encoding you can [generated in Kibana](https://elasticsearch-py_bk_2477.docs-preview.app.elstc.co/guide/en/kibana/current/api-keys.html#create-api-key).

We also need to keep the Elasticsearch link for users that don't have Kibana. This all could have been done in a future pull request

Reply:
Changed to include both methods